### PR TITLE
Add option to not run simulations when converting snapshots

### PR DESCRIPTION
**Pull request by Felixmil**
To test this:

```
devtools::load_all()

convertSnapshot(getTestDataFilePath("test_snapshot.json"),
                format = "project",
                runSimulations = FALSE)

file.rename("test_snapshot.pksim5",
            "test_snapshot_nosim.pksim5")


convertSnapshot(getTestDataFilePath("test_snapshot.json"),
                format = "project",
                runSimulations = TRUE)

file.rename("test_snapshot.pksim5",
            "test_snapshot_sim.pksim5")

```

### CODE REVIEW

**Comment by Felixmil** [comment]
!!! To merge only once [this](https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/2969) PR is merged into PKSim

**Comment by rwmcintosh** [comment]
> !!! To merge only once [this PR](https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/2969) is merged into PKSim

You should run the update dll script to get the CI build versions before merging.

File: R/snapshots.R [hunk 1]
```diff
@@ -124,12 +132,21 @@ convertSnapshot <- function(..., format, output = ".") {
...
     msg_failed = "An error occured while converting files"
   )
 
-  rSharp::callStatic("PKSim.R.Api", "RunSnapshot", SnapshotRunOptions)
+  tryCatch(
+    {
+      invisible(rSharp::callStatic("PKSim.R.Api", "RunSnapshot", SnapshotRunOptions))
+    },
+    error = function(e) {
+      message <- stringr::str_extract(as.character(e), "(?<=Message: )[^\\n]*")
```
Comment (rengelke) [inline R/snapshots.R:145]: Some error messages won't have a message line of format "Message: *". In this case `str_extract()` returns in NA resulting in 
```
Error in `convertSnapshot()`:
! NA
✖ An error occured while converting files
```

**Review by rengelke — APPROVED** [review]
Looks good, just addes a small comment FYI

File: R/snapshots.R (hunk 1 above)
Comment (Felixmil) [reply R/snapshots.R:145]: Good catch, I added an if case to handle this.

**Review by rengelke — APPROVED** [review]

**Review by rengelke — APPROVED** [review]

**Pull request merged**